webpack.config.js

```diff
@@ -22,13 +22,6 @@ webpackConfig.resolve = {
     "devtools/client/shared/vendor/react": "react",
     "devtools/client/shared/vendor/react-dom": "react-dom",
     "Services": path.join(__dirname, "node_modules/devtools-modules/client/shared/shim/Services"),
-
```

Comment:
ironically i can't kill the react react-dom mapping above because reps in devtools-modules needs them :)